Framework/PythonInterface/plugins/functions/Bessel.py

```diff
@@ -3,8 +3,6 @@
 # Copyright &copy; 2018 ISIS Rutherford Appleton Laboratory UKRI,
 #     NScD Oak Ridge National Laboratory, European Spallation Source
 #     & Institut Laue - Langevin
-# SPDX - License - Identifier: GPL - 3.0 +
```

Comment:
This isn't the section I was referring to. This definitely needs to stay.

I mean the author and date comments below.